researchstudio-sat/webofneeds
# Feat  cascading usecasegroups #2741

fixes #2741 -> Enables cascading UseCaseGroups:
fixes #2743 -> Plan a X works with Personas now
fixse #2650 -> rent/find rehearsal room usecases are also within music now

HowToTest:
- try creating some useCases
- try if #2743 is fixed
- #2741 -> cycling and lunch interest/plan should now be within "activities" useCase -> this is our first cascading useCaseGroup
